tests/unit/CoreVideoTest.php

```diff
@@ -14,50 +25,46 @@ public function setUp(): void {
 		$this->post_id = wp_insert_post(
 			[
 				'post_title'   => 'Post Title',
-				'post_content' => preg_replace(
-					'/\s+/',
-					' ',
-					trim(
-						'
-							<!-- wp:video {"id":1636} -->
-							<figure class="wp-block-video"><video autoplay loop poster="http://mysite.local/wp-content/uploads/2023/05/pexels-egor-komarov-14420089-scaled.jpg" preload="auto" src="http://mysite.local/wp-content/uploads/2023/07/pexels_videos_1860684-1440p.mp4" playsinline></video></figure>
-							<!-- /wp:video -->
-							'
-					)
-				),
+				'post_content' => '',
 				'post_status'  => 'publish',
 			]
 		);
+
+		\WPGraphQL::clear_schema();
 	}
 
 	public function tearDown(): void {
 		// your tear down methods here
 		parent::tearDown();
 
 		wp_delete_post( $this->post_id, true );
+
+		\WPGraphQL::clear_schema();
 	}
 
-	public function test_retrieve_core_video_attributes() {
-		$this->markTestSkipped( 'must be revisited since the test is failing on the CI for an unknown reason' );
-		$query  = '	
+	public function query(): string {
+		return '
 			fragment CoreVideoBlockFragment on CoreVideo {
 				attributes {
 						align
 						anchor
 						autoplay
+						className
+						lock
 						tracks
 						muted
 						caption
 						preload
 						src
+						style
 						playsInline
 						controls
 						loop
 						poster
 						id
 					}
 				}
-			
+		
 			query GetPosts {
```

Comment:
Use our query/fragment syntx from the other tests. (i.e. querying for a specific post with a passed `$variable` ID )

Reply:
Updated ✅